docs/core/additional-tools/index.md

```diff
@@ -12,7 +12,12 @@ This section compiles a list of tools that support and extend the .NET Core func
 
 ## [WCF Web Service Reference Tool](wcf-web-service-reference-guide.md)
 
-The WCF Web Service Reference is a Visual Studio connected service provider that made its debut in [Visual Studio 2017 version 15.5](https://www.visualstudio.com/news/releasenotes/vs2017-relnotes#WCFTools). This tool retrieves metadata from a web service in the current solution, on a network location, or from a WSDL file, and generates a .NET Core compatible source file containing Windows Communication Foundation (WCF) client proxy code that you can use to access the web service.
+The WCF (Windows Communication Foundation) Web Service Reference is a Visual Studio connected service provider that made its debut in [Visual Studio 2017 version 15.5](https://www.visualstudio.com/news/releasenotes/vs2017-relnotes#WCFTools). This tool retrieves metadata from a web service in the current solution, on a network location, or from a WSDL file, and generates a source file compatible with .NET Core, defining a WCF proxy class with methods that you can use to access the web service operations.
+
+## [WCF dotnet-svcutil Tool](dotnet-svcutil-guide.md)
+
+The WCF (Windows Communication Foundation) dotnet-svcutil tool is a .NET Core CLI tool that retrieves metadata from a web service on a network location or from a WSDL file, and generates a source file compatible with .NET Core, defining a WCF proxy class with methods that you can use to access the web service operations. 
+The **dotnet-svcutil** tool is an alternative option to the WCF Web Service Reference Visual Studio connected service provider. However, as a .NET Core CLI tool, the **dotnet-svcutil** tool can be run across platforms like Linux and MacOS in addition to Windows.
```

Comment:
Change to something like ""The \`dotnet-svcutil\` tool is an alternative option to the [**WCF Web Service Reference**](/dotnet/core/additional-tools/wcf-web-service-reference-guide) Visual Studio connected service provider which first shipped with Visual Studio 2017 v15.5. The \`dotnet-svcutil\` tool, as a .NET Core CLI tool, is available cross-platform on Linux, MacOS, and Windows.".